modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaUpdateCommerceAddressCheck.java

```diff
@@ -0,0 +1,141 @@
+import com.liferay.petra.string.StringPool;
+import com.liferay.petra.string.StringUtil;
+import com.liferay.source.formatter.check.util.SourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class UpgradeJavaUpdateCommerceAddressCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java")) {
+			return content;
+		}
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		List<String> importNames = javaClass.getImportNames();
+
+		if (!importNames.contains(
+				"com.liferay.commerce.model.CommerceAddress")) {
+
+			return content;
+		}
+
+		return _replaceVariableUpdateCommerceAddress(content, javaClass);
+	}
+
+	private boolean _isCommerceAddress(
+		String content, String fileContent, String variable) {
+
+		String variableTypeName = getVariableTypeName(
+			content, fileContent, variable);
+
+		return variableTypeName.equals("CommerceAddress");
+	}
+
+	private String _newUpdateCommerceAddressImplementation(
+		String variable, String indent) {
+
+		String[] methods = {
+			"getCommerceAddressId", "getName", "getDescription", "getStreet1",
+			"getStreet2", "getStreet3", "getCity", "getZip", "getRegionId",
+			"getCountryId", "getPhoneNumber", "getType"
+		};
+
+		StringBundler sb = new StringBundler(7);
+
+		sb.append(StringPool.OPEN_PARENTHESIS);
+
+		for (String method : methods) {
+			sb.append(StringPool.NEW_LINE);
+			sb.append(indent);
+			sb.append(StringPool.TAB);
+			sb.append(variable);
+			sb.append(StringPool.PERIOD);
+			sb.append(method);
+			sb.append("(),");
+		}
+
+		sb.append(StringPool.NEW_LINE);
+		sb.append(indent);
+		sb.append(StringPool.TAB);
+		sb.append("ServiceContextThreadLocal.getServiceContext()");
+		sb.append(StringPool.NEW_LINE);
+		sb.append(indent);
+		sb.append(StringPool.CLOSE_PARENTHESIS);
+
+		return sb.toString();
+	}
+
+	private String _replaceVariableUpdateCommerceAddress(
+		String content, JavaClass javaClass) {
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			Matcher matcher = _pattern.matcher(javaMethodContent);
+
+			if (!matcher.find() ||
+				!_isCommerceAddress(
+					javaMethodContent, content, matcher.group(2))) {
+
+				continue;
+			}
+
+			String line = matcher.group();
+
+			String newLine = StringUtil.replace(
+				line, matcher.group(1),
+				_newUpdateCommerceAddressImplementation(
```

Comment:
@michaelmdelcavalcanti, don't forget: methods should start their names with verbs. Please rename to `getNewUpdateCommerceAddressImplementation`.